content/en/monitors/guide/scoping_downtimes.md

```diff
@@ -1,72 +1,134 @@
+| `env:dev`, `env:demo`, `service:web-store`                                       | No (missing `env:prod`) |
+| `env:prod`, `env:demo`, `service:synthesizer`                                    | No (missing `service:web-store`) |
 
-### Two multi alert monitors, each with one "group by" scope, have the `downtime:true` monitor tag in common
+
+### Mute monitors by the union of tags
+
+To combine multiple tags into a more complex scope, use `OR` unions and avoid creating multiple Downtimes. For instance, yo would like to mute all notifications that relate to your development and staging environment. Do do so, use `env:(dev OR staging)` as your scope query.
+
+| Monitor Group                                                                    | Muted |
+| -----------                                                                      | ----  |
+| `env:dev`, `service:web-store`                                                   | Yes |
+| `env:prod`, `env:dev`, `service:synthesizer`,                                    | Yes |
+| `env:dev`, `env:staging`, `service:web-store`                                    | Yes |
+| `env:prod`, `env:demo`, `service:synthesizer`                                    | No |
+| `env:demo`, `service:synthesizer`                                                | No (missing both `env:prod` and `service:web-store`) |
+
+The union of different tags (e.g. `env:dev OR service:web-store`) is not supported. For such cases, you need to create a separate Downtime for each tag.
+
+### Mute monitors by wildcard scopes
+
+Running large upgrades within your infrastructure is not uncommon. Downtimes can help muting all impacted entities, without much extra scripting. For instance, you could be upgrading all hosts of a given service. These hosts could follow certain naming conventions in your organisation, e.g. being prefixed with their related application. Consequently, you could have hundreds of hosts named something like `host:mydemoapplication-host-1`, `host:mydemoapplication-host-2`, and so forth.
+
+To avoid muting all hosts separately, simply create a Downtime scoped by `host:mydemoapplication-*`. This will match all hosts that are prefixed accordingly. If you convention does the inverse, meaning it ends host names with their related application, can inverse the scope above as well, leading to: `host:*-mydemoapplication`.
+
+### Exclude groups from being muted
+
+If you are running your application and infrastructure on multiple environments, you likely have (at least) one production environment. All other environments might be build for testing, regression checks, or demo purposes. If you would like to avoid receiving alerts for non-production environments, you could set up a Downtime that is scopes as: `env:* -env:prod`. This scope first targets all alerts that have the `env` tag set and then excludes your production environment as a secondary step.
```

Comment:
```suggestion
If you are running your application and infrastructure on multiple environments, you likely have one production environment and multiple non-production environments (for example, testing, regression checks, or demo). To avoid receiving alerts for non-production environments, you could set up a downtime that is scoped with: `env:* -env:prod`. This scope targets all alerts that have the `env` tag set and then excludes your production environment as a secondary step.
```